code/modules/uplink/uplink_items/job.dm

```diff
@@ -16,14 +16,20 @@
 	limited_stock = 1 //please don't spam deadchat
 	surplus = 5
 
-/datum/uplink_item/role_restricted/bureaucratic_error_remote
-	name = "Organic Resources Disturbance Inducer"
-	desc = "A device that raises hell in organic resources indirectly. Single use."
-	cost = 2
+/datum/uplink_item/role_restricted/bureaucratic_error
+	name = "Organic Capital Disturbance virus"
+	desc = "Randomizes job positions presented to new hires. May lead to too many/too few security officers and/or clowns. Single use."
+	item = /obj/effect/gibspawner/generic
+	surplus = 0
 	limited_stock = 1
-	item = /obj/item/devices/bureaucratic_error_remote
+	cost = 2
+	progression_minimum = 5 MINUTES
+	restricted = TRUE
 	restricted_roles = list(JOB_HEAD_OF_PERSONNEL, JOB_QUARTERMASTER)
-	surplus = 5
```

Comment:
why'd you nuke the surplus value to 0? i don't think it's a bad change and this is already a Balance PR but why'd you do it

Reply:
ah okay then